src/PresentationalComponents/CreateImageWizard/WizardStepReview.js

```diff
@@ -96,7 +96,7 @@ const WizardStepReview = (props) => {
     return (
         <>
             { (Object.keys(props.uploadAWSErrors).length > 0 ||
-               Object.keys(props.subscriptionErrors).length > 0) &&
+               !props.isSubscriptionValid) &&
```

Comment:
The property passed from the wizard is `isValidSubscription` no?

Reply:
Fixed.